libs/components/src/lib/switch/switch.scss

```
@@ -0,0 +1,67 @@
+@use '../../../../../dist/libs/tokens/scss/tokens.constants' as constants;
+
+@use '../../../../shared/src/lib/sass/mixins/connotation/config' with (
+	$connotations: accent cta alert,
+	$shades: primary primary-text contrast,
+	$default: accent,
+);
+@use '../../../../shared/src/lib/sass/mixins/connotation' as connotation;
+
+@use '../../../../shared/src/lib/sass/mixins/appearance/config' as appearance-config with (
+	$appearances: filled ghost,
```

```suggestion
	$appearances: filled,
```
where are you using the ghost?